druntime/src/core/stdcpp/string.d

```diff
@@ -343,7 +343,7 @@ extern(D):
         ///
         inout(T)* data() inout @safe                                        { return _Get_data()._Myptr; }
         ///
-        inout(T)[] as_array() scope return inout nothrow @trusted           { return _Get_data()._Myptr[0 .. _Get_data()._Mysize]; }
+        inout(T)[] as_array() return scope inout nothrow @trusted           { return _Get_data()._Myptr[0 .. _Get_data()._Mysize]; }
```

Comment:
```suggestion
        inout(T)[] as_array() inout nothrow @trusted           { return _Get_data()._Myptr[0 .. _Get_data()._Mysize]; }
```
Now this one needs updating either to `return ref scope`, or just leave out the entire annotation since it's inside a template so it's inferred from the return statement.

Reply:
Hmm I thought I had already made that change! (to return ref scope)